src/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@
 #    Copyright (C) 2020 Chris Rizzitello                                      #
 #    Copyright (C) 2020 John Pritchard                                        #
 #    Copyright (C) 2021 Julian Xhokaxhiu                                      #
+#    Copyright (C) 2021 Cosmos                                                #
```

Comment:
If you didn't change anything here, please consider removing this line, thanks :)

Reply:
Sorry I forgot to remove this line when I deleted the other changes, it has been deleted now.